framework/dora_api/startup.py

```diff
@@ -43,15 +41,15 @@ async def startup():
     register_routers(app)
     await SqlAlchemyPersistenceContext.test(app)
 
+    app.run('localhost', 5170, app.config.get('DEBUG'), use_reloader=False) # TODO: appsettings
```

Comment:
does this make the api run on port 5170?

Reply:
yeah, didn't want to use the default port 5000 as i think many things do use that as a default,

also means it should never randomly change on us